Pycloud Deployer/config.py

```diff
@@ -0,0 +1,30 @@
+import os
+import logging
+from dotenv import load_dotenv
+
+logger =  logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('%(levelname)s - %(asctime)s - %(message)s')
+
+# Configure console log output with desired format (formatter)
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(formatter)
+logger.addHandler(console_handler)
+
+# Configure file log output to app.log with desired format
+file_handler = logging.FileHandler('app.log')
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
+
+logger.info("Loading environment variables from .env file")
+try:
```

Comment:
It might be beneficial to also verify that the required environment variables (SUBSCRIPTION_ID, LOCATION) are actually set. You could add a check after loading the environment variables to ensure these keys are present and log an appropriate message if not.